Source/Assets/PlayFabEditorExtensions/Editor/Scripts/Panels/PlayFabEditorSDKTools.cs

```diff
@@ -3,6 +3,7 @@
 using System.IO;
 using UnityEditor;
 using UnityEngine;
+using System.Reflection;
```

Comment:
That line could be moved up a little, to join the nice group of System usings above :)